skyportal/utils/http.py

```diff
@@ -1,6 +1,9 @@
 def serialize_requests_request(request):
     if request.body is not None:
-        body = request.body.decode()
+        if isinstance(request.body, bytes):
```

Comment:
What's happening here?

Reply:
just checking if needs to be decoded or not.

Reply:
Yes, but why would it ever arrive as anything other than bytes?

Reply:
Ok will remove it

Reply:
I presume it was added for a reason; just trying to understand why!

Reply:
Presumably some bug I introduced in initial testing.

Reply:
Well that broke the tests, so something about it is important I guess

Reply:
Hrm, it is used by sedm, lt, ztf, and lco without this switch.  Why would swift be different?

Reply:
Looks like the failures are unrelated?

Reply:
I am not sure why. something about how the Swift folks are returning requests?